File: C#/1893 Fases da Lua.cs
using System;
class HelloWorld {
  static void Main() {
    string[] s = Console.ReadLine().Split(' ');
    int a = int.Parse(s[0]); int b = int.Parse(s[1]);
    if (b <= 2) Console.WriteLine("nova");
    else if (b >= 3 && b <= 96 && b > a) Console.WriteLine("crescente");
    else if (b >= 97 && b <= 100) Console.WriteLine("cheia");
    else Console.WriteLine("minguante");
  }
}

File: C++/1893 Fases da Lua.cpp
#include <iostream>
using namespace std;
int main() {
    int a, b;
    cin >> a >> b;
    if (b <= 2) cout<<"nova\n";
    else if (b >= 3 && b <= 96 && b > a) cout<<"crescente\n";
    else if (b >= 97 && b <= 100) cout<<"cheia\n";
    else cout<<"minguante\n";
    return 0;
}
